lib/datadog/core/remote/transport/http.rb

```diff
@@ -120,6 +120,10 @@ def default_headers
               # Add container ID, if present.
               container_id = Datadog::Core::Environment::Container.container_id
               headers[Datadog::Core::Transport::Ext::HTTP::HEADER_CONTAINER_ID] = container_id unless container_id.nil?
+              # Pretend that stats computation are already done by the client
```

Comment:
I don't have much background here, but I think it would be helpful to extend this comment and add an explanation on what this header is used for, and why do we set it to `yes` manually.

Reply:
Thanks, I added more context in the comment